README.md

```diff
@@ -26,30 +26,30 @@ Developed by Dave Coleman, Andy McEvoy, and Mike Lautman at [PickNik Consulting]
 > Note: this package has not been released yet
 
 ```
-sudo apt-get install ros-kinetic-moveit-grasps
+sudo apt-get install ros-melodic-moveit-grasps
 ```
 
 ### Install From Source
 
 Clone this repository into a catkin workspace, then use the rosdep install tool to automatically download its dependencies. Depending on your current version of ROS, use:
 
-Kinetic:
+Melodic:
 ```
-rosdep install --from-paths src --ignore-src --rosdistro kinetic
+rosdep install --from-paths src --ignore-src --rosdistro melodic
 ```
 
 ## Robot-Agnostic Configuration
 
-You will first need a configuration file that described your robot's end effector geometry. Currently an example format can be seen in this repository at [config_robot/baxter_grasp_data.yaml](https://github.com/PickNikRobotics/moveit_grasps/blob/kinetic-devel/config_robot/baxter_grasp_data.yaml). See the comments within that file for explanations.
+You will first need a configuration file that described your robot's end effector geometry. Currently an example format can be seen in this repository at [config_robot/baxter_grasp_data.yaml](https://github.com/ros-planning/moveit_grasps/blob/melodic-devel/config_robot/baxter_grasp_data.yaml). See the comments within that file for explanations.

```

Comment:
My bad but this should be switched to `panda_grasp_data.yaml` in both the melodic and kinetic branches

Reply:
I think we should remove these instructions and only link the tutorial https://github.com/ros-planning/moveit_tutorials/pull/302.